app/operators/totalsegmentator_operator.py

```diff
@@ -34,7 +34,7 @@ def compute(self, op_input: InputContext, op_output: OutputContext, context: Exe
             os.makedirs(nii_seg_output_path)
 
         # Run TotalSegmentator
-        subprocess.run(["TotalSegmentator", "-i", nii_input_file, "-o", nii_seg_output_path])
+        # subprocess.run(["TotalSegmentator", "-i", nii_input_file, "-o", nii_seg_output_path])
```

Comment:
This should not be commented out for production MAP.

Reply:
un-uncommented